solr/core/src/java/org/apache/solr/core/PluginInfo.java

```diff
@@ -63,29 +63,42 @@ public PluginInfo(String type, Map<String, String> attrs, NamedList initArgs, Li
 
   /** class names can be prefixed with package name e.g: my_package:my.pkg.Class
    * This checks if it is a package name prefixed classname.
-   * the return value has first = package name & second = class name
    */
-  static Pair<String,String > parseClassName(String name) {
-    String pkgName = null;
-    String className = name;
-    if (name != null) {
-      int colonIdx = name.indexOf(':');
-      if (colonIdx > -1) {
-        pkgName = name.substring(0, colonIdx);
-        className = name.substring(colonIdx + 1);
+
+  public static class ClassName {
```

Comment:
Find a better name for this class?

Reply:
I didn't like the name either. Suggestions welcome